glossary.md

```diff
@@ -116,9 +146,13 @@ Global Transaction Identifiers (GTIDs) are unique transaction IDs used in MySQL
 
 ## H
 
+### Hotspot
+
+Hotspot refers to the phenomenon where the read and/or write workloads of TiKV are concentrated on one or several regions or nodes, which might cause performance bottlenecks and prevent optimal performance. To solve hotspot issues, see [Troubleshoot Hotspot Issues](/troubleshoot-hot-spot-issues.md).
```

Comment:
```suggestion
Hotspot refers to a situation where the read and write workloads in TiKV are concentrated on one or a few Regions or nodes. This can lead to performance bottlenecks, preventing optimal system performance. To solve hotspot issues, see [Troubleshoot Hotspot Issues](/troubleshoot-hot-spot-issues.md).
```